<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.mop.qa.test.regression.Test_AFL_Native_App_Regression" tests="1" time="7.470" ignored="0" failures="0" timestamp="22 May 2019 16:44:43 GMT" skipped="0" hostname="LT104209" errors="1">
  <testcase name="AFL_Upcoming_Match" time="7.470" classname="com.mop.qa.test.regression.Test_AFL_Native_App_Regression">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.mop.qa.pageobject.AFL_MatchesPage.init_elementsXpath(AFL_MatchesPage.java:90)
	at com.mop.qa.pageobject.AFL_MatchesPage.<init>(AFL_MatchesPage.java:21)
	at com.mop.qa.test.regression.Test_AFL_Native_App_Regression.AFL_Upcoming_Match(Test_AFL_Native_App_Regression.java:30)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at com.mop.qa.testrunner.TestRunner.main(TestRunner.java:167)
]]>
    </error>
  </testcase> <!-- AFL_Upcoming_Match -->
</testsuite> <!-- com.mop.qa.test.regression.Test_AFL_Native_App_Regression -->
